pkg/agent/datamodel/gpu_components_test.go

```diff
@@ -0,0 +1,24 @@
+// pkg/agent/datamodel/config_test.go
+package datamodel
+
+import (
+	"testing"
+)
+
+func TestLoadConfig(t *testing.T) {
+	// The configuration is loaded during package initialization
+	if NvidiaCudaDriverVersion == "" {
+		t.Error("NvidiaCudaDriverVersion is empty")
+	}
+	if NvidiaGridDriverVersion == "" {
+		t.Error("NvidiaGridDriverVersion is empty")
+	}
+
+	if AKSGPUCudaVersionSuffix == "" {
+		t.Error("NvidiaCudaDriverVersion is empty")
+	}
+
+	if AKSGPUGridVersionSuffix == "" {
+		t.Error(("AKSGPUGridVersionSuffix is empty"))
+	}
+}
```

Comment:
can we add some tests for checking that the driver version and suffix is parsed correctly from components.json?

Reply:
Added regex matching for parsed image driver version and suffix to ensure it's parsed properly.